docs/first-steps/policy-enforcement.md

```diff
@@ -75,10 +322,10 @@ exceptionList = new Array();
 
 //Create an allowlist using EventType, ResourceNamespace, and ResourceName combination.
 //Below is an exception example for kube-system workloads that need to run as root
-exceptionList["runAsRootAllowed"] = [
+exceptionList["runAsRootAllowed"] = new Array(
   "kube-system/cert-manager",
-  "kube-system/external-dns",
-];
+  "kube-system/external-dns"
+);
```

Comment:
I don't think you want `new Array`--not really idiomatic JS (and not 100% sure it works w/ our engine)